### DISCUSSION TITLE
return err when build cuda and cudnn on mac

### DISCUSSION THREAD

**Discussion by Xiaoaier-Z-L** [tensorchord/envd#1144] [General]
envd version:
![image](https://user-images.githubusercontent.com/96805673/199429395-90a4b5c1-598f-446d-a23a-4d81d76145ec.png)

build.envd
![image](https://user-images.githubusercontent.com/96805673/199427469-6073a8e9-74b3-45a3-89db-08ec5dd80874.png)

host: 
mac

preproduce: 
#envd up
#error: GPU is required but nvidia container runtime is not installed, please refer to https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker

Question:
What it happened when i installed cudnn and cuda with envd, why it needs nvidia container runtime? and i must build it on gpu machine?
 
Thanks!

**Comment by gaocegege** [comment]
Did you run envd up or envd build?

**Reply by Xiaoaier-Z-L** [reply]
envd up

**Comment by lalawuu** [comment]
for a workaround, you could use `envd up no-gpu` or just `envd build`

**Reply by Xiaoaier-Z-L** [reply]
![image](https://user-images.githubusercontent.com/96805673/199430174-349f0594-3e76-44d9-b051-4506b561d08e.png)

**Reply by Xiaoaier-Z-L** [reply]
when i use envd up no-gpu --force, it works!

**Comment by Xiaoaier-Z-L** [comment]
Maybe, i should only build one image with cuda&cudnn and do not need launch one container, or use envd up no-gpu --force
